charts/csi-secrets-store-provider-aws/templates/daemonset.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "sscdpa.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+{{ include "sscdpa.labels" . | indent 2 }}
+spec:
+  updateStrategy:
+{{ toYaml .Values.updateStrategy | indent 4 }}
+  selector:
+    matchLabels:
+      app: {{ template "sscdpa.name" . }}
+  template:
+    metadata:
+{{ include "sscdpa.labels" . | indent 6 }}
+{{- if .Values.podLabels }}
+{{- toYaml .Values.podLabels | nindent 8 }}
+{{- end }}
+{{- if .Values.podAnnotations }}
+      annotations:
+{{- toYaml .Values.podAnnotations | nindent 8 }}
+{{- end }}
+    spec:
+      {{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+{{ toYaml .Values.imagePullSecrets | indent 8 }}
+      {{- end }}
+      serviceAccountName: {{ template "sscdpa.serviceAccountName" . }}
+      hostNetwork: true
+      containers:
+        - name: provider-azure-installer
```

Comment:
I belive will make more sense to have aws instead of azure here

Reply:
Done

Reply:
I work with both providers and sometimes it happens to get confused 😛